specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json

```diff
@@ -2256,6 +2754,94 @@
         }
       }
     },
+    "CertificateResource": {
+      "description": "Certificate resource payload.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/CertificateProperties",
+          "description": "Properties of the certificate resource payload.",
+          "x-ms-client-flatten": false
+        }
+      }
+    },
+    "CertificateProperties": {
+      "description": "Certificate resource payload.",
+      "required": [
+        "vaultUri",
+        "keyVaultCertName"
+      ],
+      "type": "object",
+      "properties": {
```

Comment:
>properties [](start = 7, length = 10)

In the example, it appears that a lot of these properties come from the Key Vault cert and are effectively read only. Can you add `"readOnly": true` declarations on all such properties in the swagger?

Reply:
sure.